internal/parser/simple_parser_rules.go

```diff
@@ -51,23 +51,37 @@ func (p *simpleParser) parseSQLStatement(r reporter) (stmt *ast.SQLStmt) {
 	case token.KeywordCommit:
 		stmt.CommitStmt = p.parseCommitStmt(r)
 	case token.KeywordCreate:
-		p.parseCreateStmt(stmt, r)
+		p.parseCreateStmts(stmt, r)
 	case token.KeywordDelete:
-		stmt.DeleteStmt = p.parseDeleteStmt(r)
+		p.parseDeleteStmts(stmt, nil, r)
 	case token.KeywordDetach:
 		stmt.DetachStmt = p.parseDetachDatabaseStmt(r)
+	case token.KeywordDrop:
+		p.parseDropStmts(stmt, r)
 	case token.KeywordEnd:
 		stmt.CommitStmt = p.parseCommitStmt(r)
+	case token.KeywordInsert:
+		stmt.InsertStmt = p.parseInsertStmt(nil, r)
+	case token.KeywordReindex:
+		stmt.ReIndexStmt = p.parseReIndexStmt(r)
```

Comment:
If we renamed `stmt.ReIndex`, I think the keyword should also be renamed. Make sure to run `go generate` after that, for the sake of keyword scanning, since that is generated.

Reply:
Wow this had a lot of repurcussions across scanner.
Fixed. 
Will mention in the PR.